tests/oadp/datamover/test_vsr.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+
+import pytest as pytest
+
+from oadp_resources.oadp.datamover.volume_snapshot_restore import VolumeSnapshotRestore
+
+path = str(Path(__file__).parent.parent.parent)
+
+
+@pytest.fixture(scope='session')
+def vsr_obj():
+    yaml_path = os.path.join(path, "sample_input/vsr.yaml")
+    vsr = VolumeSnapshotRestore(yaml_file=yaml_path)
+    vsr.create()
+    yield vsr
+    vsr.delete()
+
+
+def test_vsr_get(vsr_obj):
+    obj = list(vsr_obj.get())[0].instance
+    expected_name = 'vsr-v69qs'
```

Comment:
vsr-for-test?

Reply:
wdym?

Reply:
done:
https://github.com/oadp-qe/openshift-adp-python-wrapper/compare/2af0548dadb4e2d09cd43927b1149b228c81faf1..de92a4e33e60946201b177dab723c5a9acc7446e